.travis.yml

```diff
@@ -113,11 +113,10 @@ script:
     - export PYTHONPATH=/tmp/$TRAVIS_JOB_ID/lib/python$TRAVIS_PYTHON_VERSION/site-packages
     # install easybuild-framework to unique temporary location using prepared sdist tarball
     - mkdir -p $PYTHONPATH; easy_install --prefix /tmp/$TRAVIS_JOB_ID $TRAVIS_BUILD_DIR/dist/easybuild-framework*tar.gz
-    # actively break "import setuptools" and "import pkg_resources",
+    # make sure there are no "import setuptools" or "import pkg_resources" statements,
     # since EasyBuild should not have a runtime requirement on setuptools
-    - echo 'raise ImportError("setuptools should no longer be relied on")' > /tmp/$TRAVIS_JOB_ID/setuptools.py
-    - echo 'raise ImportError("setuptools is not available")' > /tmp/$TRAVIS_JOB_ID/pkg_resources.py
-    - export PYTHONPATH=/tmp/$TRAVIS_JOB_ID:$PYTHONPATH
+    - SETUPTOOLS_IMPORTS=$(egrep -RI '^(from|import) pkg_resources|^(from|import) setuptools' easybuild/ || true)
+    - test "x$SETUPTOOLS_IMPORTS" = "x" || (echo "Found setuptools imports in easybuild/:\n${SETUPTOOLS_IMPORTS}" && exit 1)
```

Comment:
"Found setuptools and/or pkg_resources..."

Reply:
Since `pkg_resources` is provided  by `setuptools`, technically this is correct, but I see the point, so I'll tweak it, thanks.